test/fixtures.rb

```diff
@@ -1,27 +1,56 @@
+# rubocop:disable Layout/TrailingWhitespace
```

Comment:
Hmm I don't believe a heredoc should trigger this I will double check on this though.

Reply:
I don’t believe too, but rubocop do it